.evergreen/test-plugin.sh

```diff
@@ -26,13 +26,13 @@ _authToken=$NPM_AUTH_TOKEN
 EOT
 
 cat .npmrc
-npm install
+npm install --registry https://registry.npmjs.org/
```

Comment:
I'm was doing a drive by clean-up while addressing another ci issue and noticed this whole file is not used and can go away, we can keep this change, but is not really required to change it